src/components/application_manager/include/application_manager/application.h

```diff
@@ -443,6 +443,17 @@ class Application : public virtual InitialApplicationData,
   virtual bool is_navi() const = 0;
   virtual void set_is_navi(bool allow) = 0;
 
+  /**
+   * @brief Returns is_rc
```

Comment:
@BSolonenko unclear description. Please update it

Reply:
@BSolonenko also please add `@return` line with description

Reply:
@AKalinich-Luxoft  Fixed in https://github.com/smartdevicelink/sdl_core/pull/2048/commits/89790bbec52b1c03e17e863389ee538708ebf277